File: 8_multilevel_inheritance.cpp
#include<bits/stdc++.h>
using namespace std;
class A
{
    public:
        void displayA()
        {
            cout<<"Capital letter A"<<endl;
        }
};
class B: public A // class B class A er thake inherit kortaca mane class B er modddhe A er sob property ase
{
    public:
        void displayB()
        {
            cout<<"Capital letter B"<<endl;
        }
};
class C: public B // class C class B er thake inherit kortaca mane class C er modddhe B er sob property ase. ak kotahi class C er moddhe class A ebong class B ei 2 ta class er property ei ase
{
    public:
        void displayC()
        {
            cout<<"Capital letter C"<<endl;
        }
};
int main()
{
    C obj;
    obj.displayA();
    obj.displayB();
    obj.displayC();
    return 0;
}

File: 8_multilevel_inheritance.py
class A: # base class (grandparent)
    def __init__(self) -> None:
        pass

    def displayA(self):
        print("Capital letter A")

class B(A): # derive class 1 (parent)
    def __init__(self) -> None:
        pass

    def displayB(self):
        print("Capital letter B")

class C(B): # derive class 2 (child)
    def __init__(self) -> None:
        pass

    def displayC(self):
        print("Capital letter C")


obj = C()
obj.displayA()
obj.displayB()
obj.displayC()
# multilevel inheritance: kono derived class jodi base class er derived class thake property inherit kore tokon take multilevel inheritance bola jai.
#  a class is derived from a class that is also derived from another class.